en/docs/SetupAndInstall/DeployingWSO2APIManager/DistributedDeployment/deploying-wso2-api-m-in-a-distributed-setup.md

```diff
@@ -579,14 +579,14 @@ This section involves setting up the API Store node and enabling it to work with
             revoke_endpoint = "https://[API-Gateway-LB-Host-or-IP]:${https.nio.port}/revoke"
         ```
 
-3.  Optionally, configure High Availability (HA) for the Store.
+3.  Optionally, configure High Availability (HA) for the Dev Portal.
 
     !!! warning
-        This is **ONLY applicable** if you need to configure **HA for the Store.**
+        This is **ONLY applicable** if you need to configure **HA for the Dev Portal.**
```

Comment:
Shall we change Dev Portal to Developer Portal in all the places in descriptions?

Reply:
Fixed